CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 4.3.0
+
+- Implements Client#close method, allows Infinity for "http2SessionIdleTime" parameter
```

Comment:
How is `Infinity` useful? The maximum transaction time is 10 minutes. Waiting any longer than that is just waiting because you can.

Reply:
@faunaee Hi, Ewan! This one is related to the HTTP/2 session lifetime. The `Infinity` value means that the user is responsible for terminating the fauna `Client` instance and it won't be terminated automatically. Check [this PR](https://github.com/fauna/faunadb-js/pull/471/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R245) for more details

Reply:
Thanks @tigger9flow ! That makes sense.